### DISCUSSION TITLE
Time input for Oven

### DISCUSSION THREAD

**Discussion by ablyes** [ekutner/home-connect-hass#262] [General]
I can't see any timer input for the ove. 
By default it's one minute, that means when i start the oven from remove, it's just for one unique minute. 
I didn't find any input for the time, this is why i'm here, at this place, to ask if this timer input exists in this alternative ?
Thank you. 

Oven réf : Neff B57CS24H0

**Comment by sjoerdvdd** [comment]
Is this resoveld yet? i'm having similar troubles with setting the temperature and time and not having the right step size and limits in both numerical displays 

<img width="385" alt="image" src="https://github.com/user-attachments/assets/e47f62ce-46f8-4362-979f-c463e2be216c">
